eth/protocols/eth/protocol.go

```diff
@@ -39,11 +40,11 @@ const ProtocolName = "eth"
 
 // ProtocolVersions are the supported versions of the `eth` protocol (first
 // is primary).
-var ProtocolVersions = []uint{ETH68}
+var ProtocolVersions = []uint{ETH69, ETH68}
 
 // protocolLengths are the number of implemented message corresponding to
 // different protocol versions.
-var protocolLengths = map[uint]uint64{ETH68: 17}
+var protocolLengths = map[uint]uint64{ETH68: 17, ETH69: 15}
```

Comment:
This is wrong, though the number of useful messages indeed goes down, the max offset is still 17 (i.e. the last message's "byte").

Reply:
You'd need to "compact" the message id space to reduce this, but that's a lot of work for no real gain, so it was never pushed down when removing message types, only ever up.